content/en/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods.md

```diff
@@ -24,9 +26,10 @@ vacated by the evicted critical add-on pod or the amount of resources available
 
 ### Marking pod as critical
 
-To be considered critical, the pod has to run in the `kube-system` namespace (configurable via flag) and
+Prior to v1.11, critical pod has to run in the `kube-system` namespace, this restriction was removed after v1.11 and pod in any namespace can be configed as a critical pod by the following either way:
 
-* Have the priorityClassName set as "system-cluster-critical" or "system-node-critical", the latter being the highest for entire cluster. Alternatively, you could add an annotation `scheduler.alpha.kubernetes.io/critical-pod` as key and empty string as value to your pod, but this annotation is deprecated as of version 1.13 and will be removed in 1.14.
+* Ensure the PodPriority [feature gates](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) is enabled, have the priorityClassName set as "system-cluster-critical" or "system-node-critical", the latter being the highest for entire cluster. Available in v1.10+
 
+* Alternatively, ensure both PodPriority and ExperimentalCriticalPodAnnotation feature gates are enabled, you could add an annotation `scheduler.alpha.kubernetes.io/critical-pod` as key and empty string as value to your pod, but this annotation is deprecated as of version 1.13 and will be removed in a future release.
```

Comment:
the annotation was planned to remove from 1.14, but seems that it still available in the current latest version. Guessing it may removed in the future.

Reply:
this annotation appears available [since v1.4](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.4.md#changelog-since-v148), do I need to make a statement on that?

Reply:
OK,1.4 is too old. I believe it's no necessary to add this.